src/btree/bt_vrfy.c

```diff
@@ -240,14 +240,18 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
         goto done;
 
     /*
-     * Get a list of the checkpoints for this file. Empty objects have no checkpoints, in which case
-     * there's no work to do.
+     * Get a list of the checkpoints for this file. Empty objects and ingest tables have no
+     * checkpoints, in which case there's no work to do.
      */
     WT_ERR_NOTFOUND_OK(__wt_meta_ckptlist_get(session, name, false, &ckptbase, NULL), true);
     if (ret == WT_NOTFOUND) {
         ret = 0;
         goto done;
-    }
+    } else if (WT_SUFFIX_MATCH(name, ".wt_ingest"))
+        WT_ERR_MSG(session, EBUSY,
```

Comment:
In all the cases when the logic is broken, we return WT_ERROR instead of EBUSY. I understand that in this cases it would be turned into WT_ERROR later, but I think it's simpler to set it to WT_ERROR straight away since it's a rules violation.

Reply:
done b2c1f095425ceee63ea2df9ab43039e59625ca48